app/ts/components/App.tsx

```diff
@@ -1,10 +1,47 @@
-import { Route, Router } from './Router.js'
-import { SendEthPage } from './SendEthPage.js'
+import { Route, Router, useParams } from './HashRouter.js'
 
 export function App() {
 	return (
 		<Router>
-			<Route path='/' component={SendEthPage} />
+			<Route path='/#!/tx/send' component={Send} />
+			<Route path='/#!/tx/:transaction_hash' component={Transaction} />
```

Comment:
The plan is to drop the `!` right?  Can we also drop the extraneous `/` and do `#send` and `#tx/:transaction_hash` or something?  It feels very noisy having all of those slashes in there.

Reply:
oh it already works with or without the hash bang as long as valid hash👌just updated the example